File: qlty-cli/tests/lang/java/basic.in/Returns.java
class Returns {
    void f0() {
    }

    void f1() {
        return;
    }

    void f2() {
        if (true) {
            return;
        } else {
            return;
        }
    }

    void f3() {
        if (true){
            return;
        } else if (true) {
            return;
        } else {
            return;
        }
    }

    void f4() {
        if (true){
            return;
        } else if (true) {
            return;
        } else if (true) {
            return;
        } else {
            return;
        }
    }
}

File: qlty-cli/tests/lang/csharp/basic.in/Returns.cs
using System;

public class Returns
{
    public void F0()
    {
    }

    public void F1()
    {
        return;
    }

    public void F2()
    {
        if (true)
        {
            return;
        }
        else
        {
            return;
        }
    }

    public void F3()
    {
        if (true)
        {
            return;
        }
        else if (true)
        {
            return;
        }
        else
        {
            return;
        }
    }

    public void F4()
    {
        if (true)
        {
            return;
        }
        else if (true)
        {
            return;
        }
        else if (true)
        {
            return;
        }
        else
        {
            return;
        }
    }
}
